setuptools/msvc.py

```diff
@@ -1038,9 +1038,13 @@ def VCTools(self):
             tools += [os.path.join(si.VCInstallDir, path)]
 
         elif self.vs_ver >= 15.0:
-            host_dir = (
-                r'bin\HostX86%s' if self.pi.current_is_x86() else r'bin\HostX64%s'
-            )
+            if self.pi.current_is_x86():
+                host_dir = r'bin\HostX86%s'
+            elif self.pi.current_cpu == 'arm64':
+                host_dir = r'bin\HostARM64%s'
+            else:
+                host_dir = r'bin\HostX64%s'
```

Comment:
Even better than an if/else statement would be something like:

```suggestion
            host_dir = os.path.join('bin', f'Host{self.host_id.upper()}%s')
```

And then of course implement the property `host_id` to reflect the correct string to insert. In general, it's good practice to DRY - avoid repeating the assignment and prefix and suffix of the value.

Reply:
Adding a property for something that's only used once seemed a bit of overkill.  Let me know if what I did works.

Reply:
I took it one step further in 3218d666e49245b0878db8b19ba20d8cd72d6424.